src/System.Net.Http/tests/FunctionalTests/DiagnosticsTests.cs

```diff
@@ -488,7 +488,7 @@ public void SendAsync_ExpectedDiagnosticSourceNewAndDeprecatedEventsLogging()
                     diagnosticListenerObserver.Enable();
                     using (var client = new HttpClient())
                     {
-                        var response = client.GetAsync(Configuration.Http.RemoteEchoServer).Result;
+                        client.GetAsync(Configuration.Http.RemoteEchoServer).Result.Dispose();
```

Comment:
thanks!

There are other tests in the same class that never dispose response. I can fix other tests